scripts/e2e_smoke.py

```
@@ -0,0 +1,269 @@
+    graph = build_graph()
+    initial = AgentState(user_input=BUILD_PROMPT)
+    context = SimContext(epw_path=EPW, output_dir=OUTPUT_DIR)
+    config = RunnableConfig(configurable={"thread_id": "e2e_build"})
+
+    try:
+        final = run_session(
+            graph, initial, context, config,
+            on_interrupt=_auto_approve,
+            on_event=_on_event,
+        )
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)